sara r5 support

andresv opened the pull request:
This PR adds support for SARA-R5.

Depends on:
* https://github.com/BlackbirdHQ/atat/pull/128
* https://github.com/BlackbirdHQ/ublox-sockets/pull/11

andresv committed:
increase cert upload response timeout to 3s

sara-r5 needs longer time

andresv committed:
Cargo.toml: add feature: sara-r5

andresv committed:
add sara-r5 support

andresv committed:
update to: embedded-hal 1.0.0-alpha.8

andresv committed:
set atat and ublox-sockets to git versions

andresv committed:
add features: logger-log and logger-defmt

before this change `logger-defmt` was named `defmt-impl`

andresv committed:
README: add sara-r5 and update logging docs

MathiasKoch commented:
Could you fix the conflicting file, and set atat and ublox-sockets to their newly released alpha-8 versions?

atat: 0.17.0
ublox-sockets: 0.4.0

andresv committed:
update atat and ublox-sockets

andresv committed:
Merge branch 'master' into sara-r5

andresv committed:
keep using defmt-impl to turn on defmt logging

defmt-impl seems to be more common feature name, now we also have log-impl to turn on log based logging

andresv committed:
fix logger comments

MathiasKoch commented:
Looks great!
Thanks you for the contribution :+1:
